packages/array-max/src/index.js

```diff
@@ -3,39 +3,46 @@ import isArray from 'is-any-array';
 /**
  * Computes the maximum of the given values
  * @param {Array<number>} input
- * @param {int} [options.from] - Start index (inclusive) for the slice within which we look for the max
- * @param {int} [options.to] - End index (exclusive) for the slice within which we look for the max
+ * @param {number} [options.fromIndex] - Start index (inclusive) for the slice within which we look for the max
+ * @param {number} [options.toIndex] - End index (exclusive) for the slice within which we look for the max
  * @return {number}
  */
 export default function max(input, options = {}) {
-  let { from, to } = options;
-
   if (!isArray(input)) {
     throw new TypeError('input must be an array');
   }
 
   if (input.length === 0) {
     throw new TypeError('input must not be empty');
   }
+  let { fromIndex, toIndex } = options;
 
-  if (from === undefined) {
-    from = 0;
-  } else if ((from < 0) | (from > input.length)) {
+  if (fromIndex === undefined) {
+    fromIndex = 0;
+  } else if (
+    (fromIndex < 0) |
+    (fromIndex > input.length) |
+    !Number.isInteger(fromIndex)
+  ) {
     throw new Error(
-      'start index must be greater than 0 and smaller equal than length',
+      'start index must be a integer greater than 0 and smaller equal than length',
     );
   }
 
-  if (to === undefined) {
-    to = input.length;
-  } else if ((to <= from) | (to > input.length)) {
+  if (toIndex === undefined) {
+    toIndex = input.length;
+  } else if (
+    (toIndex <= fromIndex) |
+    (toIndex > input.length) |
+    !Number.isInteger(toIndex)
+  ) {
     throw new Error(
-      'to index must be greater equal than from index and smaller equal than length',
+      'toIndex index must be a integer greater equal than fromIndex index and smaller equal than length',
```

Comment:
```suggestion
      'toIndex must be an integer greater than fromIndex and at most equal to length',
```

Reply:
wow! many thanks for your time and careful review!